test/e2e/test/http_client.go

```diff
@@ -29,7 +29,8 @@ func NewHTTPClient(caCerts []*x509.Certificate) *http.Client {
 		certPool.AddCert(c)
 	}
 	transport.TLSClientConfig = &tls.Config{
-		RootCAs: certPool,
+		MinVersion: tls.VersionTLS12, // ES 6.8 only supports 1.2
```

Comment:
I think this is also useful just to make sure we aren't doing something silly, but is definitely not necessary. The default is TLS 1.0.